source/val/validate_extensions.cpp

```diff
@@ -2226,13 +2226,36 @@ spv_result_t ValidateExtInst(ValidationState_t& _, const Instruction* inst) {
             ValidateOperandDebugType(_, "Base Type", inst, 5, ext_inst_name);
         if (validate_base_type != SPV_SUCCESS) return validate_base_type;
         for (uint32_t i = 6; i < num_words; ++i) {
-          CHECK_OPERAND("Component Count", SpvOpConstant, i);
+          bool invalid = false;
           auto* component_count = _.FindDef(inst->word(i));
-          if (!_.IsIntScalarType(component_count->type_id()) ||
-              !component_count->word(3)) {
+          if (component_count->opcode() == SpvOpConstant) {
+            if (!_.IsIntScalarType(component_count->type_id()) ||
```

Comment:
This isn't checking the 32- or 64-bit size requirement. Didn't before either, but that is spelled out in the spec.